lib/vistle/core/messages.h

```diff
@@ -435,10 +435,11 @@ class V_COREEXPORT AddObjectCompleted: public MessageBase<AddObjectCompleted, AD
     port_name_t m_orgDestPort;
 };
 
-//! connect an output port to an input port of another module
-class V_COREEXPORT Connect: public MessageBase<Connect, CONNECT> {
+//! Base class for connect and disconnect
+
```

Comment:
probably won't work portably due to unclear memory layout with multiple inheritance